acceptance/openstack/client_test.go

```diff
@@ -12,6 +16,18 @@ import (
 	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
 )
 
+const (
+	backupSuffix = ".backup"
+	tmpl         = `
```

Comment:
You can move it to `var` and store it as `[]bytes`, instead of converting it each time